jest.config.base.js

```diff
@@ -0,0 +1,21 @@
+const { TextDecoder, TextEncoder } = require("util");
+
+module.exports = {
+  roots: ["<rootDir>/src"],
+  transform: {
+    "^.+\\.ts$": "ts-jest",
+  },
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+  collectCoverage: true,
```

Comment:
Reading the docs, this says the code coverage tracker it could significantly slow down the tests - is that the case for us? https://jestjs.io/docs/configuration#collectcoverage-boolean

Reply:
@JST5000 I'm not sure, but we need code coverage reports yes?

Reply:
Probably - but I don't know that we necessarily need it on every test. (If it's a significant slowdown, we could have it only included in CI for example using the commandline arg instead of the config setting. If it's a couple seconds it probably doesn't matter though.)

Reply:
I don't think it'll slow down the tests too much, since the Explorer runs tests in CI with coverage and they're still decently speedy.